python/hsfs/feature_group.py

```diff
@@ -640,6 +648,27 @@ def commit_delete_record(
         """
         self._feature_group_engine.commit_delete(self, delete_df, write_options)
 
+    def compute_statistics(self):
```

Comment:
This is wrong. `compute_statistics` should be a member of the `FeatureGroupBase` class so that it can be used also by the on-demand feature groups